src/devices/Si7021/Si7021.cs

```diff
@@ -22,20 +22,20 @@ public void Dispose()
         {
         }
 
-        public double ReadTempFahrenheit()
+        public double ReadTemperatureInFahrenheit()
         {
-            double temp_celcius = ReadTempCelcius();
-            double temp_fahrenheit = (ReadTempCelcius() * (9 / 5)) + 32;
+            double temp_celcius = ReadTemperatureInCelcius();
```

Comment:
Same here, I think that if we use `tempCelcius` it goes more along to what we do in the rest of the repo.

Reply:
sure, will change